### ISSUE TITLE
Docs feedback: /clients/aws/ecs/_index.md

### ISSUE THREAD

**Issue by gaffneyc** [grafana/loki#5271]
I feel like the docs should be updated to recommend the `amazon/aws-for-fluent-bit:stable` docker image now that Fluent Bit includes a Loki output. Also, while trying to figure out the latest version tag for grafana/fluent-bit-plugin-loki it looks like a new container is built for every commit to [Loki](https://hub.docker.com/r/grafana/fluent-bit-plugin-loki/tags) which makes finding a tagged version annoying.

Here is the equivalent config for firelens to use the built in Fluent Bit output. One of the main issues is that there is no config for the URL so it needs to be split into pieces (host, port, tls, user, password). Also, the label keys need to be prefixed with `$`. 

```json
"logConfiguration": {
  "logDriver": "awsfirelens",
  "options": {
    "Name": "loki",
    "host": "logs-prod-us-central1.grafana.net",
    "port": "443",
    "http_user": "12345",
    "http_passwd": "API_KEY",
    "labels": "env=staging",
    "tls": "on",
    "remove_keys": "container_id,ecs_task_arn",
    "label_keys": "$container_name,$ecs_task_definition,$source,$ecs_cluster"
  }
}
```

Firelens router task:

```json
{
  "essential": true,
  "image": "amazon/aws-for-fluent-bit:stable",
  "name": "log_router",
  "firelensConfiguration": {
    "type": "fluentbit"
  },

  "memoryReservation": 50,

  "logConfiguration": {
    "logDriver": "awslogs",
    "options": {
      "awslogs-group": "firelens",
      "awslogs-region": "us-east-1",
      "awslogs-create-group": "true",
      "awslogs-stream-prefix": "firelens"
    }
  }
}
```

**Comment by Ingwersaft** [comment]
Yeah, I was very confused and wasted nearly a whole day getting this to work :/

**Comment by DekusDenial** [comment]
When I used the official loki output plugin from fluent bit by AWS (amazon/aws-for-fluent-bit) I have gotten way too many broken bad chunks and logs were basically lost as the fluent bit pods kept failing then in crashloop. I had to switch to use loki’s own plugin here and the issue disappeared without any change in the conf. Just my experience to share and also wonder why that was the case.

To me, I was not confused about the doc but reluctant to use loki’s version of the plugin at first until the above issue surfaced and I could not fix it. At that point it had already taken me almost half a day before it just took me couple minutes to make the switch and everything started to behave correctly.

But yes a lot of image tags as it’s happening per commit but if you look up by released version it’s not that hard but this should be documented in some way possible.
